src/imageproxy.rs

```diff
@@ -192,6 +193,42 @@ impl TryFrom<ImageProxyConfig> for Command {
     }
 }
 
+
+///Trying
+#[derive(Debug, serde::Deserialize)]
+pub struct LayersInfo {
+    ///Placeholder
+    pub Success: String,
+    ///Layers
+    pub Layers: Vec<BlobInfo>
+}
+
+///Algo
+#[derive(Debug, serde::Deserialize)]
+pub struct Algorithm {
+    name: String,
+    mime: String, 
+    prefix: Vec<u8>,
+    decompressor: String,
+    compressor:  String
+}
+
+///BlobInfo
+#[derive(Debug, serde::Deserialize)]
+pub struct BlobInfo {
+    ///Digest
+    pub Digest: String,
+    ///Size
+    pub Size: i64,
+    URLs: Option<Vec<String>>,
+    Annotations: Option<HashMap<String, String>>,
+    pub MediaType: oci_spec::image::MediaType,
+    CompressionOperation: i32,
+    CompressionAlgorithm: Option<Algorithm>, 
+    CryptoOperation: i32
```

Comment:
If we're not actually using a field, I'd probably comment it out for now
```suggestion
    // CryptoOperation: i32
```